PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -2230,21 +2247,24 @@ class PurchasesTests: XCTestCase {
     func testAttributionDataSendsNetworkAppUserId() {
         let data = ["yo": "dog", "what": 45, "is": ["up"]] as [String: Any]
 
-        Purchases.deprecated.addAttributionData(data, from: AttributionNetwork.appleSearchAds, forNetworkUserId: "newuser")
+        Purchases.deprecated.addAttributionData(data,
+                                                from: AttributionNetwork.appleSearchAds,
+                                                forNetworkUserId: "newuser")
 
         setupPurchases()
 
         expect(self.backend.invokedPostAttributionData).toEventually(beTrue())
+        let maybeInvokedMethodParams = self.backend.invokedPostAttributionDataParameters
```

Comment:
I think we could do an `XCTUnwrap` here, since we're assuming that it's non-nil for all the checks afterwards

Reply:
I don't think we intended to make it valid for `maybeInvokedMethodParams`, it looks like it's just a bug in the test

Reply:
same for the next test

Reply:
sure! makes sense 👍🏼